runtime/rasdump/javadump.cpp

```diff
@@ -1484,6 +1484,93 @@ JavaCoreDumpWriter::writeEnvUserArgsHelper(J9VMInitArgs *vmArgs)
 				_OutputStream.writeCharacters("\n");
 			}
 		}
+
+		/* Ignored options for -Xjit and -Xaot follows the following rules:
+		 * 1. If Xjit is followed by -Xint or -Xnojit,
+			then any -Xjit:optionString beforehand appears as ignored
+		 * 2. If XX:+MergeCompilerOptions is present,
+			then all Xjit options after Xint or Xnojit should not be ignored
+		 * 3. If multiple Xjit string appear, all but the last one will appear as ignored
+		 *
+		 * and the same for -Xaot
```

Comment:
Please
* spell options completely (`-Xjit` instead of `Xjit`)
* the second and subsequent lines of a multi-line comment should start with `*`
* end each sentence with a period
```c
		/* Ignored options for -Xjit and -Xaot follows these rules:
		 * 1. If -Xjit is followed by -Xint or -Xnojit,
		 *    then any -Xjit:optionString beforehand appears as ignored.
		 * 2. If -XX:+MergeCompilerOptions is present,
		 *    then all -Xjit options after -Xint or -Xnojit should not be ignored.
		 * 3. If multiple -Xjit string appear, all but the last one will appear as ignored.
		 *
		 * Similar rules apply for -Xaot.
		 */
```